src/core/store/modules/app/app.actions.ts

```diff
@@ -117,6 +125,47 @@ const getUserAppData = createAsyncThunk<void, { network: Network; route: Route;
   }
 );
 
+/* -------------------------------------------------------------------------- */
+/*                                  Services                                  */
+/* -------------------------------------------------------------------------- */
+
+const checkServicesStatus = createAsyncThunk<void, void, ThunkAPI>(
+  'app/checkServicesStatus',
+  async (_arg, { dispatch, extra }) => {
+    const { YEARN_ALERTS_API } = extra.config;
+    try {
+      const servicesStatusResponse = await get(`${YEARN_ALERTS_API}/health`);
+      if (servicesStatusResponse.status !== 200) throw new Error('Service status provider not currently accesible');
```

Comment:
typo, it's `accessible`

Reply:
fixed https://github.com/yearn/yearn-finance-v3/pull/674/commits/ac85e1e489ab9ec7bbd482f5d227a5a682bdcd01 😁